cross/get_iplayer/Makefile

```diff
@@ -0,0 +1,27 @@
+PKG_NAME = get_iplayer
+
+NAME = get_iplayer
+PKG_DIST_SITE = git://git.infradead.org/get_iplayer.git
```

Comment:
http://www.infradead.org/get_iplayer/html/get_iplayer.html mentions  ftp://ftp.infradead.org/pub/get_iplayer/ as the download site.